utils/test/garden-test-utils.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { render, configure } from 'react-testing-library';
+import { ThemeProvider } from '../../packages/theming/src';
+
+configure({ testIdAttribute: 'data-test-id' });
+
+const LtrProvider = ({ children }) => {
+  return <ThemeProvider>{children}</ThemeProvider>;
+};
+
+LtrProvider.propTypes = {
+  children: PropTypes.node
+};
+
+const RtlProvider = ({ children }) => {
+  return <ThemeProvider rtl>{children}</ThemeProvider>;
+};
+
+RtlProvider.propTypes = {
+  children: PropTypes.node
+};
+
+const customLtrRender = (ui, options) => render(ui, { wrapper: LtrProvider, ...options });
+const customRtlRender = (ui, options) => render(ui, { wrapper: RtlProvider, ...options });
```

Comment:
Just curious why under `utils` instead of the `testing` package? Is this not considered generally useful for other components that render under our theme provider?

Reply:
Since this is the first implementation of this testing library I don't think it's ready for us to be shared as a consumable for other teams. Also, no other teams use `react-testing-library` at this time.

I'd like to have us use this in some more advanced scenarios before we decide it's stable enough to share.